### PR TITLE
Navigates to MY TRACKS.

### PR DESCRIPTION
**Pull request by asaikarthikeya**
### Description
#553
When any user checks his Statistics of total tracks,distance,duration. It is more convenient to check the details of each track by clicking on the statistics layout rather than accessing it through bottomnavigationview.

https://user-images.githubusercontent.com/70392921/103418398-77bcf200-4bb4-11eb-8a59-4e30f0c39a72.mp4

### CODE REVIEW

**Comment by asaikarthikeya** [comment]
@arvindnegi1 please review .

File: org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java [hunk 1]
```diff
@@ -495,6 +494,12 @@ protected void onGPSIndicatorClicked() {
         getActivity().startActivity(intent);
     }
 
+    @OnClick(R.id.cardView2)
+    protected void Go_to_MyTracks(){
```
Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java:498]: use camelCase for function name

Comment (arvindnegi1) [inline org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java:498]: use camelCase for function name

**Review by arvindnegi1 — CHANGES_REQUESTED** [review]

**Comment by arvindnegi1** [comment]
@asaikarthikeya please do required changes.
As told you in mail. Follow Java Naming conventions.

**Comment by asaikarthikeya** [comment]
I changed the name using name conventions but did not squash the commits(My
bad) will do that soon.
Grateful to receive your support. thank you.

**Pull request closed without merging**